ChechpointStyles.css

```diff
@@ -0,0 +1,78 @@
+
+body { background-color: silver;
```

Comment:
suggestion: let's try to format CSS based on community best practices like this:

```css
body {
  background-color: silver;
}
```

Also, check [this CSS style guide](http://udacity.github.io/frontend-nanodegree-styleguide/css.html) from Udacity for more reference.